Magento_Theme/styles/modules/_copyright.scss

```diff
@@ -0,0 +1,7 @@
+.copyright {
+    background-color: $gray-light;
+    color: $white;
+    text-align: center;
+    font-size: $font-size-extra-small;
+    padding: 10px 0;
+}
```

Comment:
I didn't figure out a name for this, but are we planning to add some Utilities like (typography, spacing) additional to component ?